applications/SolidMechanicsApplication/custom_processes/assign_scalar_field_to_entities_process.hpp

```diff
@@ -39,7 +37,7 @@ class AssignScalarFieldToEntitiesProcess : public Process
     /// Pointer definition of AssignScalarFieldToEntitiesProcess
     KRATOS_CLASS_POINTER_DEFINITION(AssignScalarFieldToEntitiesProcess);
 
-    enum EntityType { NODES, CONDITIONS, ELEMENTS};
```

Comment:
Is this an enum class?

Reply:
I assume now is defined in the base class

Reply:
Yes, changed in the base class to enum class